src/js/edu-benefits/components/benefits-eligibility/BenefitsSelectionFields.jsx

```diff
@@ -20,11 +22,31 @@ export default class BenefitsSelectionFields extends React.Component {
               options={relinquishableBenefits}
               value={this.props.data.benefitsRelinquished}
               onValueChange={(update) => {this.props.onStateChange('benefitsRelinquished', update);}}/>
+          {showRelinquishedEffectiveDate(this.props.data.benefitsRelinquished.value)
+            ? <DateInput required={showRelinquishedEffectiveDate(this.props.data.benefitsRelinquished.value)}
+                errorMessage="Please provide a response"
+                validation={validateIfDirtyDateObj(this.props.data.benefitsRelinquishedDate, isValidDateField)}
```

Comment:
It's a required element to indicate when you're benefits you're currently receiving will get shot off in favor of the new benefits through CH 33.